test/integration/config.spec.js

```diff
@@ -47,6 +47,28 @@ describe('config', function() {
       expect(js, 'to equal', json);
     });
 
+    it('should rethrow error from absolute path configuration', function() {
```

Comment:
The error messages of both tests still contain:
`Error: Cannot find module 'test\\integration\\fixtures\\config\\mocharcWithThrowError.js'`
This shouldn't be, right?

Reply:
no, sorry. I'm writing nonsense.